src/analysis/scripts/export_connectivity.sh

```diff
@@ -95,8 +124,28 @@ then
         # Export neighborhood_ways as SHP
         ec_export_table_shp "${OUTPUT_DIR}" "neighborhood_ways"
 
+        # Export census blocks
+        ec_export_table_shp "${OUTPUT_DIR}" "neighborhood_census_blocks"
+        ec_export_table_geojson "${OUTPUT_DIR}" "neighborhood_census_blocks"
```

Comment:
This file will be massively large for pretty much any larger analysis, and we shouldn't be displaying it on a web map anyways.

The Germantown one is 4.2mb and geojson.io struggles.